stages/test/test_cloud-init.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python3
+
+import os.path
+
+from osbuild.testutil.imports import import_module_from_path
+
+
+# Test that the 'datasource_list' option is dumped as a single line
+# in the configuration file.
+def test_cloud_init_datasource_list_single_line(tmp_path):
+    treedir = os.path.join(tmp_path, "tree")
+    config_dir = f"{treedir}/etc/cloud/cloud.cfg.d/"
+    os.makedirs(config_dir, exist_ok=True)
+
+    stage_path = os.path.join(os.path.dirname(__file__), "../org.osbuild.cloud-init")
```

Comment:
(super duper nitpick, sorry!) most of the other tests do the stage setup as the thing as it's boring and repetitive (and I wonder now if we should create a pytest fixture for it). It does not really matter and you don't need to change it but I wanted to mention it as I like symmetry :)

Reply:
I implemented the fixture and in addition and while I was at it, I also implemented a fixture for loading the stage schema (this was also duplicated all over the place). PTAL